src/modules/dtfs-storage/dtfs-storage-file.service.ts

```diff
@@ -0,0 +1,40 @@
+import { FileGetPropertiesResponse, RestError, ShareFileClient, StorageSharedKeyCredential } from '@azure/storage-file-share';
+import { Inject, Injectable } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import DtfsConfig from '@ukef/config/dtfs-storage.config';
+
+import { DtfsStorageException } from './exception/dtfs-storage.exception';
+import { DtfsStorageFileNotFoundException } from './exception/dtfs-storage-file-not-found.exception';
+
+@Injectable()
+export class DtfsStorageFileService {
+  private readonly baseUrl: string;
+  private readonly storageSharedKeyCredential: StorageSharedKeyCredential;
+
+  constructor(
+    @Inject(DtfsConfig.KEY)
+    { baseUrl, accountName, accountKey }: ConfigType<typeof DtfsConfig>,
+  ) {
+    this.baseUrl = baseUrl;
+    this.storageSharedKeyCredential = new StorageSharedKeyCredential(accountName, accountKey);
+  }
+
+  getFileProperties(fileName: string, fileLocationPath: string): Promise<FileGetPropertiesResponse> {
+    const shareFileClient = this.getShareFileClient(fileName, fileLocationPath);
+
+    return shareFileClient.getProperties().catch((error) => this.handleGetFilePropertiesError(error, fileName, fileLocationPath));
+  }
+
+  private getShareFileClient(fileName: string, fileLocationPath: string): ShareFileClient {
```

Comment:
As mentioned in a huddle -- We might want to abstract this out into a `dtfs-storage-client` or similar to allow for similar unit testing to `graph-client-service`... but this can wait until we need it (later on in this story)